src/data/shortlist.h

```diff
@@ -183,13 +184,12 @@ class LexicalShortlistGenerator : public ShortlistGenerator {
                             Ptr<const Vocab> srcVocab,
                             Ptr<const Vocab> trgVocab,
                             size_t srcIdx = 0,
-                            size_t trgIdx = 1,
+                            size_t /*trgIdx*/ = 1, // not used in this class
```

Comment:
(no need for the comment I think. Not important.)

Reply:
Comment removed.